docs/multimedia.md

```diff
@@ -1635,6 +1635,10 @@ JavDB 有多个备用域名，本路由默认使用永久域名 <https://javdb.c
 
 </Route>
 
+### 作者文章
+
+\<Route author="JamYiz" example="/sehuatang/user/411096"path="/sehuatang/user/:uid":paramsDesc="\['uid, 必选 - 用户 uid, 可在用户主页 URL 中找到 ']"selfhost="1"/>
```

Comment:
```suggestion
<Route author="JamYiz" example="/sehuatang/user/411096" path="/sehuatang/user/:uid" :paramsDesc="['用户 uid, 可在用户主页 URL 中找到']" selfhost="1"/>
```
- Every attribute needs to be separated by a space.
- No need to mention the path parameters and the necessity again.
- Since you've set `selfhost` attribute, you need to update the docs on what envs need to be set in `install/README.md`.

docs: https://docs.rsshub.app/joinus/new-rss/add-docs.html